src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx

```diff
@@ -91,7 +91,7 @@ const AddTaskModal = (props) => {
     {value:"Other",label:"Other"}
   ]
   const [projectCategory, setProjectCategory] = useState('');
```

Comment:
are we using this anywhere , if not you can remove them..

Reply:
removed ..works as expected.. didn't test hours remaining two points working as expected.